package.json

```diff
@@ -11,7 +11,7 @@
     "clean": "turbo clean",
     "format": "prettier --write .",
     "format-check": "prettier --check .",
-    "all-check": "pnpm install && pnpm format && pnpm format-check && pnpm lint && pnpm test && pnpm build"
```

Comment:
I initially swapped it because I found it strange to do checking if the step before we did the actual formatting.

Not sure how folks are using this, but I am typically running this to ensure everything will pass in the CI/CD later. By just doing format checking, if there are formatting errors, the dev will have to run `pnpm format` manually and re-run `pnpm all-check`. Given the formatting takes as long as checking, I figured we should just do the fixes. But maybe that isn't best 🤷

Reply:
Ah, maybe you're right. For some reason, I thought `prettier --write` (which is what `pnpm format` calls) only fixed what it could, but might not be able to fix everything. But now I'm realizing I was incorrect.